client/utils/helpers.js

```diff
@@ -0,0 +1,60 @@
+import jwtDecode from 'jwt-decode';
+
+export const summarize = (str, wordCount = 17) => str
```

Comment:
The function name here isn't descriptive of the action it performs. It seems like it `truncates` a string of characters and I think that `truncate` would be a better function name.